spec/lib/haml/filters/growstuff_markdown_spec.rb

```diff
@@ -60,4 +74,27 @@ def output_link(crop, name=nil)
     rendered.should match "<a href=\"http://example.com\">test</a>"
   end
 
+  it 'converts quick member links' do
+    @member = FactoryGirl.create(:member)
+    rendered = Haml::Filters::GrowstuffMarkdown.render(input_member_link(@member.login_name))
+    rendered.should match /#{output_member_link(@member)}/
+  end
+
+  it "doesn't convert nonexistent members" do
+    rendered = Haml::Filters::GrowstuffMarkdown.render(input_member_link("not a member"))
+    rendered.should match /not a member/
+  end
+
+  it 'converts @ member links' do
+    @member = FactoryGirl.create(:member)
+    rendered = Haml::Filters::GrowstuffMarkdown.render("@#{@member.login_name}")
+    rendered.should match /#{output_member_link(@member, "@#{@member.login_name}")}/
+  end
+
+  it "doesn't convert nonexistent @ members" do
+    rendered = Haml::Filters::GrowstuffMarkdown.render("@not-a-member")
```

Comment:
This isn't a legal member name. It's probably worth testing "invalidly-formatted name" and "valid but unused name" separately.